core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/extensions.scala

```diff
@@ -150,3 +150,5 @@ case class ArrayAccess(array: Expression, index: Expression) extends Expression
 case class JsonAccess(json: Expression, path: Seq[String]) extends Expression {}
 case class Collate(string: Expression, specification: String) extends Expression {}
 case class Iff(condition: Expression, thenBranch: Expression, elseBranch: Expression) extends Expression {}
+
+case class Over(input: Expression, partitionBy: Seq[Expression], orderBy: Seq[SortOrder]) extends Expression {}
```

Comment:
Seems unused 

```suggestion
```

Reply:
Ah yes. It was how I was going to do it until I saw that there was existing Ir.